src/components/QuoteComponent.jsx

```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect } from 'react';
+import style from './Home.module.css'
+import quotes from '../quotes.json';
+
+const QuoteComponent = () => {
+  const [quoteData, setQuoteData] = useState({ quote: '', author: '' });
+
+  useEffect(() => {
+    const getQuotes = async () => {
```

Comment:
This is a static import, so it doesn't need to be async and it doesn't need to be in a try/catch.  If it's missing it will fail at compile time.